tests/unit/authorizers/devise-test.js

```diff
@@ -2,17 +2,32 @@ import { describe, beforeEach, it } from 'mocha';
 import { expect } from 'chai';
 import sinon from 'sinon';
 import Devise from 'ember-simple-auth/authorizers/devise';
+import { registerDeprecationHandler } from '@ember/debug';
 
 describe('DeviseAuthorizer', () => {
   let authorizer;
   let block;
   let data;
+  let warnings;
 
   beforeEach(function() {
+    registerDeprecationHandler((message, options, next) => {
+      // in case a deprecation is issued before a test is started
+      if (!warnings) {
+        warnings = [];
+      }
+
+      warnings.push(message);
+      next(message, options);
+    });
```

Comment:
Would it make sense to move this into the test instead as it's only needed for that one test anyway (same thing applies below for the Devise test of course)?

Reply:
The handler isn't registered early enough when it's setup inside the test, but I'll see if there are other options.

Reply:
I think you'd just have to move creation of the authorizer into the test as well (or just create another one in the test)?

Reply:
👍